src/user/tipsandtricks.rst

```diff
@@ -61,6 +61,15 @@ Here is how a ``.condarc`` file would look like:
 In addition to the channel priority we recommend to always install your packages inside a new environment instead the root environment from anaconda/miniconda.
 Using envs make it easier to debug problems with packages and ensure the stability of your root env.
 
+Note: in the past ``conda-forge`` used to vendorize some of ``defaults`` dependencies that were not built in our infrastructure,
+like compilers run-times, to avoid the mixing channel problem.
+However, with the ``strict`` option, we no longer have to vendorize those (which has its own problems too),
```

Comment:
Do we now recommend everybody to use strict? If so we probably should also put that here: http://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge

Reply:
Yes, I agree.

Reply:
In `conda 4.7` where `strict` will be default and we will need to update that but I guess that, b/c mixing channels without `strict` is broken, we should add it there too.

Reply:
Done! Not sure if we should leave the comment about conda 4.7 there though.